authors/apps/comments/tests/test_comments_endpoints.py

```diff
@@ -240,3 +250,251 @@ def test_cannot_delete_reply_if_you_didnt_author_it(self):
             "You can only delete a reply you authored",
             response.data['response'])
         self.assertEqual(response.status_code, status.HTTP_403_FORBIDDEN)
+
+    def test_cannot_like_nonexistent_comment(self):
+        self.create_comment()
+        response = self.client.post(
+            reverse(
+                'comments:comment-likes', kwargs={'pk': self.comment_id + 1}),
+            content_type='application/json'
+        )
+        self.assertIn(
+            "does not exist!",
+            response.data['message'])
+        self.assertEqual(response.status_code, status.HTTP_404_NOT_FOUND)
+
+    def test_cannot_get_like_status_nonexistent_comment(self):
+        self.create_comment()
+        response = self.client.get(
+            reverse(
+                'comments:comment-likes', kwargs={'pk': self.comment_id + 1}),
+            content_type='application/json'
+        )
+        self.assertIn(
+            "does not exist!",
+            response.data['message'])
+        self.assertEqual(response.status_code, status.HTTP_404_NOT_FOUND)
+
+    def test_cannot_like_comment_you_authored(self):
```

Comment:
Great work @pbkabali especially with `get_single_object_using_pk` and notifications for liking a comment but I feel the names of your functions for testing could be more clear for instance this function `test_cannot_like_comment_you_authored ` would be clearer if you said `test_user_cannot_like_a_comment_they_authored` and many other functions for testing otherwise good work

Reply:
Thanks @oma0256 for making that point. I will take a second look at my work. I appreciate the review